### PR TITLE
feat: bottom bar

### PR DESCRIPTION
**Pull request by mserwo**

### CODE REVIEW

**Comment by GabrLit** [comment]
comment

File: src/pages/home/home.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ export const Home = () => {
   return (
     <Layout>
       <div>home</div>
```
Comment (mserwo) [inline src/pages/home/home.tsx]: super!

File: src/pages/home/home.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ export const Home = () => {
...
+      <div>tu jest mój rodzic</div>
```
Comment (GabrLit) [inline src/pages/home/home.tsx]: to be deleted, unnecessary

File: src/components/footer/bottom-bar.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+const BottomBar = () => {
+  return (
+    <div className={styles.bottomBar}>
+      <div className={styles.bottomBar_divider}></div>
+      <div className={styles.bottomBar_content}>
+        <div className={styles.bottomBar_leftText}>
+          {"Copyright © 2021 UI8 LLC. All rights reserved"}
+        </div>
+        <div className={styles.bottomBar_rightFrame}>
```
Comment (GabrLit) [inline src/components/footer/bottom-bar.tsx:11]: frame would be bad naming here. Iframe in web development refers to a specific object: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

File: src/components/footer/footer.module.scss [hunk 4]
```diff
@@ -32,3 +36,56 @@
...
 }
+
+.bottomBar {
+  display: flex;
+  padding-bottom: 32px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 32px;
+  @include fonts.poppins;
```
Comment (GabrLit) [inline src/components/footer/footer.module.scss:47]: what is this include here, include should be at the top if anything? and as far as I remember we do have a font family already specified somewhere in global scss file?

File: src/components/footer/footer.module.scss [hunk 5]
```diff
@@ -32,3 +36,56 @@
...
+    height: 1px;
+    color: colors.$neutrals6;
+  }
+  &_content {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: flex-start;
+  }
+
```
Comment (GabrLit) [inline src/components/footer/footer.module.scss:61]: be consistent, have blank line between those nested selectors or not

File: src/components/footer/footer.module.scss [hunk 6]
```diff
@@ -32,3 +36,56 @@
...
+    font-weight: 400;
+    line-height: 20px;
+  }
+  &_accept {
+    color: colors.$primary1;
+    text-align: right;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 20px;
```
Comment (GabrLit) [inline src/components/footer/footer.module.scss:89]: this should be probably defined as a mixin in typography, for example:

```ts
@mixin body1--bold {
  font-size: 24px;
  line-height: 32px;
  font-weight: 600;
}
```

see how this font is named in Figma, each one has its own name

**Review by GabrLit — CHANGES_REQUESTED** [review]

**Pull request closed without merging**